src/pivot-table/components/cells/utils/get-next-cell.ts

```diff
@@ -0,0 +1,26 @@
+import type { Cell, List } from "../../../../types/types";
+
+const getPseudoDimensionAdjustedIndex = (cell: Cell | null): number => {
+  if (!cell || !cell.parent) {
+    return 0;
+  }
+
+  if (cell.isPseudoDimension) {
+    return cell.parent.children.length - 1;
```

Comment:
what if we have only one pseudo dimension (only avgSales for example)? will this if check return -1 and then subsequently we will subtract 1 from `nextCellIndex`?